### ISSUE TITLE
Station data seems not to have been updated since August 25, 2023

### ISSUE THREAD

**Issue by dupuy** [chrnola/citibike-api-docs#1]
While the specified URL (`https://layer.bicyclesharing.net/map/v1/nyc/map-inventory`) still provides data, the `features.#.properties.station.last_reported` values seem to indicate this is no longer being updated, and the `bikes_available` and `docks_available` values don't match what is shown in the CitiBike app.

**Comment by chrnola** [comment]
Good catch! I haven't looked at this repo in a long time. My guess is that CitiBike abandoned this API when Lyft forced them to move to their tech. 😞

**Comment by dupuy** [comment]
That sounds quite plausible. Perhaps you could share the tech you used for reverse-engineering the app? If it's using HTTPS (I would hope so), it seems like you would need some way to be on the device to get the key information for decrypting the traffic.

**Comment by chrnola** [comment]
That's correct, the app communicates over TLS but I used (the incredible) [mitmproxy](https://mitmproxy.org) to intercept the plaintext.
